modules/example_pump_asset_hierarchy/transformations/pump_asset_hierarchy-load-collections_pump.yaml

```diff
@@ -0,0 +1,21 @@
+externalId: pump_asset_hierarchy-load-collections_pump
+name: pump_asset_hierarchy-load-collections_pump
+destination:
+  type: asset_hierarchy
+
+# Specify credentials separately like this:
+# You can also use different credentials for the running transformations than the ones you use to deploy
+authentication:
+  clientId: {{clientId}}
+  clientSecret: {{clientSecret}}
+  tokenUrl: {{tokenUrl}}
+  # Optional: If idP requires providing the scopes
+  cdfProjectName: {{cdfProjectName}}
+  scopes:
+    - {{scopes}}
+  # Optional: If idP requires providing the audience
+  audience: {{audience}}
+schedule:
+  # every hour
+  interval: '0 * * * *'
+  is_paused: true
```

Comment:
Just realised I have mixed camelCase and snake_case in the apm_simple transformations as well. How is this handled on the SDK side? Should we allow anything?

Reply:
I think maybe non-matching attributes are just silently ignored. If we use isPaused here, I guess it will work...

Reply:
Yes, non-matching attributes are silently ignored (to avoid breaking the SDK when new ones are added to the API). 

`is_paused` is actually missing from the SDK, so it will not be set.